python/tutorials/10-block-scaled-matmul.py

```diff
@@ -151,23 +141,7 @@ def block_scaled_matmul_kernel(  #
     offs_sn = (pid_n * (BLOCK_N // 128) + tl.arange(0, BLOCK_N // 128)) % N
 
     MIXED_PREC: tl.constexpr = ELEM_PER_BYTE_A == 1 and ELEM_PER_BYTE_B == 2
-
-    if MIXED_PREC:
-        b_desc = tl.make_tensor_descriptor(
-            b_desc_or_tensor,
-            shape=[N, K // ELEM_PER_BYTE_B],
-            strides=[K // ELEM_PER_BYTE_B, 1],
-            block_shape=[BLOCK_N, BLOCK_K // ELEM_PER_BYTE_B],
-        )
-    else:
-        b_desc = b_desc_or_tensor
-        tl.inline_asm_elementwise("prefetch.tensormap [$1]; // dummy $0", "=r,l", [b_desc], dtype=tl.int32,
-                                  is_pure=False, pack=1)
-
-    tl.inline_asm_elementwise("prefetch.tensormap [$1]; // dummy $0", "=r,l", [a_desc], dtype=tl.int32, is_pure=False,
```

Comment:
Only concession is we can't prefetch the tensormap any more because the raw pointer isn't exposed in the API. Could hack around it if this is really needed for performance though.

Reply:
I believe this was only helping the case where the descriptor was passed by pointer from the host and we won't allow this case anymore